html_css/assets/css/style.css

```diff
@@ -0,0 +1,1091 @@
+/* TEMPORARY STYLE FOR PIXEL PERFECT */
+/* .chromeperfectpixel-overlay {
+  width: 1440px !important;
+  height: initial !important;
+} */
+
+@import url(./modern-normalize.css);
+
+@font-face {
+  font-family: "Montserrat";
+  src: url("../assets/fonts/Montserrat-Regular.ttf") format("ttf");
+  font-weight: 400;
+  font-style: normal;
+  display: swap;
+}
+
+@font-face {
+  font-family: "Montserrat";
+  src: url("../assets/fonts/Montserrat-Medium.ttf") format("ttf");
+  font-weight: 500;
+  font-style: normal;
+  display: swap;
+}
+
+@font-face {
+  font-family: "Montserrat";
+  src: url("../assets/fonts/Montserrat-SemiBold.ttf") format("ttf");
+  font-weight: 600;
+  font-style: normal;
+  display: swap;
+}
+
+@font-face {
+  font-family: "Montserrat";
+  src: url("../assets/fonts/Montserrat-Bold.ttf") format("ttf");
+  font-weight: 700;
+  font-style: normal;
+  display: swap;
+}
+
+:root {
+  --main-text-color: #000000;
+  --secondary-text-color: #5C5C5C;
+  --button-bg-default: #1B63F0;
+  --button-bg-hover: #3D7FFF;
+  --button-bg-pressed: #175EE9;
+}
+
+
+body {
+  margin: 0 auto;
+  position: relative;
+  font-family: 'Montserrat', 'Verdana', 'Arial', 'Helvetica', sans-serif;
+  color: #000000;
+  max-width: 1440px;
+}
+
+h1 {
+  font-weight: 700;
+  font-size: 36px;
+  line-height: 1.22;
+  margin: 0;
+}
+
+h2 {
+  font-weight: 700;
+  font-size: 26px;
+  line-height: 1.0;
+}
+
+p {
+  font-size: 14px;
+  line-height: 1.57;
+  color: var(--secondary-text-color);
+}
+
+a {
+  text-decoration: none;
+}
+
+li {
+  list-style: none;
+}
+
+button,
+.ready__apply-btn {
+  padding: 12px 24px;
+  width: 116px;
+  height: 40px;
+  background-color: #1B63F0;
+  border-radius: 22px;
+  border: 1px;
+  cursor: pointer;
+  font-weight: 700;
+  font-size: 12px;
+  line-height: 1.33;
+  color: #FFFFFF;
+}
+
+button:hover {
+  background: #3D7FFF;
+}
+
+button:active {
+  background: #175EE9;
+}
+
+.container {
+  margin: 0 auto;
+  max-width: 1024px;
+}
+
+.bg-icons {
+  position: absolute;
+  top: 0;
+  left: 0;
+  max-width: 746px;
+  z-index: -2;
+}
+
+.bg-green {
+  position: relative;
+}
+
+.bg-green-top,
+.bg-green-bottom {
+  position: absolute;
+  top: 0;
+  right: 0px;
+}
+
+.bg-green-top {
+  z-index: 2;
+  max-width: 740px;
+}
+
+.bg-green-bottom {
+  max-width: 778px;
+}
+
+header {
+  z-index: 4;
+  padding-top: 44px;
+  margin: 0 auto;
+  max-width: 850px;
+  height: 40px;
+  display: flex;
+  flex-direction: row;
```

Comment:
Default value for display: flex. Do not to be added

Reply:
Removed